.ansible-lint

```diff
@@ -0,0 +1,13 @@
+---
+skip_list:
+  - package-latest
+  - command-instead-of-shell
+  - git-latest
+  - no-tabs
+  - no-handler
+  - var-spacing
+  - command-instead-of-module
+  - risky-shell-pipe
```

Comment:
these don't need to be excluded - see the config on ansible-proxmox [here](https://github.com/robert-sandor/ansible-proxmox/blob/main/.ansible-lint)